### PR TITLE
Some fixs

### PR DESCRIPTION
**Pull request by Majo494**
Se creo link logo. 
Se mejoro calidad logo
Se creo validacion de campos nombre y correo al enviar consulta
Se cambio imagen

### CODE REVIEW

File: app/assets/stylesheets/orders.scss [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+  color:red;
+}
+
+.error_input {
+  border-color:red;
+}
+
+.label-hide {
+  display:none;
+
```
Comment (MathiasMoralesDixtra) [inline app/assets/stylesheets/orders.scss]: Remove empty space

File: app/assets/stylesheets/orders.scss [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+}
+
+#submit-form {
+  margin-top: 10px
+}
+
+.field_with_errors { display: inline }
```
Comment (MathiasMoralesDixtra) [inline app/assets/stylesheets/orders.scss]: Fix indentation and left one line at the left of the file

File: app/assets/stylesheets/styles.css [hunk 3]
```diff
@@ -2176,6 +2178,10 @@ progress {
   border-radius: 0.25rem;
   transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
 }
+
+.input-error {
+    border-color:red
+}
```
Comment (MathiasMoralesDixtra) [inline app/assets/stylesheets/styles.css:2186]: add empty line after this

File: app/assets/stylesheets/styles.css [hunk 4]
```diff
@@ -310,6 +310,8 @@ label {
     color: #161925;
 }
 
+.field_with_errors { display: none; }
```
Comment (MathiasMoralesDixtra) [inline app/assets/stylesheets/styles.css]:
```suggestion
.field_with_errors { 
    display: none;
 }
```

File: app/views/home/_form.html.erb [hunk 5]
```diff
@@ -1,36 +1,29 @@
...
+        <%= form.text_field :name, class: @order.errors[:name].any? ? 'form-control input-error' : 'form-control', placeholder:"Enter your name...", id:"inputName" %>
+        <label for="inputName", class="<%=@order.errors[:name].any? ? 'error_explanation' : 'labelinput' %>"><%= form.label :name %></label>
     </div>
-  </div>
-</div>
\ No newline at end of file
+    <div class="form-floating mb-4">
+      <%= form.text_field :email, class: @order.errors[:email].any? ? 'form-control input-error' : 'form-control', placeholder:"name@example.com", id:"inputEmail"%>
+      <label for="inputEmail", class="<%=@order.errors[:email].any? ? 'error_explanation' : 'labelinput' %>"><%= form.label :email %></label>
+
```
Comment (MathiasMoralesDixtra) [inline app/views/home/_form.html.erb]: delete empty line

File: config/database.yml [hunk 6]
```diff
@@ -25,12 +25,4 @@ test:
 
 production:
   <<: *default
-  database: yoyakuprod
-
-db_heroku:
-  <<: *default
-  host: ec2-54-147-93-73.compute-1.amazonaws.com
```
Comment (MathiasMoralesDixtra) [inline config/database.yml:32]: Probably we have to change this data, I don't know if this is in a singular commit or not, but in the case that it is a singular commit we have to delete this and would avoid changing this information. @mauriguz @Majo494 what do you think guys?

File: app/assets/stylesheets/orders.scss (hunk 1 above)
Comment (mauriguz) [inline app/assets/stylesheets/orders.scss]:
```suggestion
```

File: app/assets/stylesheets/orders.scss (hunk 2 above)
Comment (mauriguz) [inline app/assets/stylesheets/orders.scss]:
```suggestion
.field_with_errors {
  display: inline
}

```

File: app/assets/stylesheets/styles.css (hunk 4 above)
Comment (mauriguz) [inline app/assets/stylesheets/styles.css]:
```suggestion
.field_with_errors {
  display: none;
}
```

File: app/assets/stylesheets/styles.css (hunk 3 above)
Comment (mauriguz) [inline app/assets/stylesheets/styles.css:2186]:
```suggestion
}

```

File: app/assets/stylesheets/styles.css [hunk 7]
```diff
@@ -2596,6 +2602,13 @@ textarea.form-control-lg {
   height: calc(3.5rem + 2px);
   padding: 1rem 0.75rem;
 }
+
+.form-floating > .form-control-error,
+.form-floating > .form-select {
+    height: calc(3.5rem + 2px);
```
Comment (mauriguz) [inline app/assets/stylesheets/styles.css:2611]: I see that this same calc is being used in other places, does it make sense to move it to a variable? also, what it's supposed to accomplish?

File: app/views/home/_form.html.erb [hunk 8]
```diff
@@ -1,36 +1,29 @@
...
-      </div>
+<div class="modal-header bg-gradient-primary-to-secondary p-4">
+  <h5 class="modal-title font-alt text-white" id="feedbackModalLabel">Contact Us</h5>
+  <button class="btn-close btn-close-white" type="button" data-bs-dismiss="modal" aria-label="Close"></button>
+</div>
+<div class="modal-body border-0 p-4">
+  <%= form_with model: @order, local:false do |form| %>
+    <div class="form-floating mb-4">
+        <%= form.text_field :name, class: @order.errors[:name].any? ? 'form-control input-error' : 'form-control', placeholder:"Enter your name...", id:"inputName" %>
+        <label for="inputName", class="<%=@order.errors[:name].any? ? 'error_explanation' : 'labelinput' %>"><%= form.label :name %></label>
```
Comment (mauriguz) [inline app/views/home/_form.html.erb]: any specific reason to wrap form.label in <label>? couldn't we just use form.label?

File: app/views/home/index.html.erb [hunk 9]
```diff
@@ -1,48 +1,37 @@
-<div id="page-top">
-<!-- Navigation-->
+<div id="top">
 <nav class="navbar navbar-expand-lg navbar-light fixed-top shadow-sm" id="mainNav">
   <div class="container px-5">
-    <img src="/Ico6.png"
-    <%= image_tag("Yoyaku-02 1.png", :alt => "rss feed", :class => "ico-yoyaku") %>
-    <a class="navbar-brand fw-bold" href="#page-top">Yoyaku</a>
+    <a href="#top">
```
Comment (mauriguz) [inline app/views/home/index.html.erb]: I believe we don't need this new anchor, wrapping both title and image with the existing anchor should be enough

File: app/views/home/index.html.erb [hunk 10]
```diff
@@ -159,7 +131,7 @@
     </div>
   </div>
 </section>
-<!-- Basic features section-->
+ Basic features section
```
Comment (mauriguz) [inline app/views/home/index.html.erb]: Is this intended?

File: app/views/home/index.html.erb [hunk 11]
```diff
@@ -173,125 +145,39 @@
...
       </h3>
-<!--      <h3 class="text-white display-1 lh-1 mb-4">-->
-<!--        We are Dixtra!-->
-<!--      </h3>-->
       <a class="btn btn-outline-light py-3 px-4 rounded-pill" href="https://www.dixtra.com" target="_blank">About us</a>
     </div>
   </div>
 </section>
-<!-- App badge section-->
+ App badge section
```
Comment (mauriguz) [inline app/views/home/index.html.erb]: Idem

File: app/views/home/index.html.erb [hunk 12]
```diff
@@ -173,125 +145,39 @@
...
   <div class="container px-5">
     <h2 class="text-center text-white font-alt mb-4">Get the app now!</h2>
     <div class="d-flex flex-column flex-lg-row align-items-center justify-content-center">
-<!--      <a class="me-lg-3 mb-4 mb-lg-0" href="#!"><img class="app-badge" src="assets/img/google-play-badge.svg" alt="..." /></a>-->
-<!--      <a href="#!"><img class="app-badge" src="assets/img/app-store-badge.svg" alt="..." /></a>-->
     </div>
   </div>
 </section>
-<!-- Footer-->
+ Footer
```
Comment (mauriguz) [inline app/views/home/index.html.erb]: Idem

File: config/environments/production.rb [hunk 13]
```diff
@@ -95,6 +95,9 @@
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
 
+  ActionView::Base.field_error_proc = Proc.new do |html_tag, instance|
```
Comment (mauriguz) [inline config/environments/production.rb:98]: isn't enough by adding it to config/environment?

File: app/views/home/javascript.js.erb [hunk 14]
```diff
@@ -0,0 +1,3 @@
+$(function() {
+    $('#modal-form').html('<%= j render "home/form" %>')
```
Comment (mauriguz) [inline app/views/home/javascript.js.erb:2]: this is already in the html, why do we need it in both places?

**Review by mauriguz — CHANGES_REQUESTED** [review]

File: app/views/home/_form.html.erb (hunk 5 above)
Comment (mauriguz) [reply app/views/home/_form.html.erb]:
```suggestion
```

File: config/database.yml (hunk 6 above)
Comment (mauriguz) [reply config/database.yml:32]: we'll revoke this info and move forward with other credentials, not a blocker for this pr

File: app/assets/stylesheets/styles.css (hunk 7 above)
Comment (mauriguz) [reply app/assets/stylesheets/styles.css:2611]: this file could use a full refactor, will ignore for now

File: config/environments/production.rb (hunk 13 above)
Comment (mauriguz) [reply config/environments/production.rb:98]: no need to fix in this PR

File: app/views/home/javascript.js.erb (hunk 14 above)
Comment (mauriguz) [reply app/views/home/javascript.js.erb:2]: yes it's needed

**Review by mauriguz — APPROVED** [review]

**Review by MathiasMoralesDixtra — APPROVED** [review]

**Pull request merged**